test/typeahead.spec.js

```diff
@@ -317,6 +317,19 @@ describe('Typeahead', function () {
         }, 100);
     });
 
+    it('should call onSelected function', function () {
+        const selectedFunc = jasmine.createSpy('onSelected spy');
+        $('.test-element').typeahead({getMethod: TEST_METHOD, onSelected: selectedFunc});
+
+        const typeahead = $('.typeahead');
+        let dropdown = $('.typeahead ~ ul.items.dropdown-menu');
+
+        typeahead.val(TEST_VALUE);
+        fireInputEvent(typeahead[0]);
+
+        dropdown.find('li:eq(2)').click();
+        expect(selectedFunc).toHaveBeenCalled();
```

Comment:
is there a way to check the passed parameter

Reply:
yes there is

Reply:
So can you do it?

Reply:
Do it!